src/common/libmmgtypes.h

```diff
@@ -516,7 +516,7 @@ typedef MMG5_InvMat * MMG5_pInvMat;
 typedef struct {
   MMG5_pPar     par;
   double        dhd,hmin,hmax,hsiz,hgrad,hgradreq,hausd;
-  double        min[3],max[3],delta,ls,rmc;
+  double        min[3],max[3],delta,ls,lxreg,rmc;
```

Comment:
Why have you name the variable lxreg and not xreg?

Reply:
There already is a attribute of struct info that is called xreg: an integer to tell whether the option is activated or not. Hence, xreg is always an integer, lxreg is always a double.

Reply:
Then, in other occurences, lxreg is used to keep consistency, even if there would be no conflict with the name xreg.